Cargo.toml

```diff
@@ -27,31 +27,31 @@ fs = ["async-std"]
 serde = ["serde_qs", "serde_crate", "serde_json", "serde_urlencoded", "url/serde"]
 
 [dependencies]
-fastrand = "1.4.0"
-base64 = "0.13.0"
-futures-lite = "1.11.1"
-async-channel = "1.5.1"
-infer = "0.7.0"
-pin-project-lite = "0.2.0"
-url = "2.1.1"
-anyhow = "1.0.26"
+fastrand = "2.0.1"
+base64 = "0.21.5"
+futures-lite = "2.2.0"
+async-channel = "2.1.1"
+infer = "0.15.0"
+pin-project-lite = "0.2.13"
+url = "2.5.0"
+anyhow = "1.0.79"
 
 # features: async_std
-async-std = { version = "1.6.0", optional = true }
+async-std = { version = "1.12.0", optional = true }
 
 # features: hyperium/http
-http = { version = "0.2.0", optional = true }
+http = { version = "1.0.0", optional = true }
```

Comment:
This is a breaking change requiring a major version

Reply:
The newest version on crates.io is v2.12.0, though, so the 3.0.0 in Cargo.toml was fine.

Reply:
it mostly means I will not commit to merging this change